packages/react-components/react-carousel-preview/library/package.json

```diff
@@ -46,8 +46,9 @@
     "@fluentui/react-utilities": "^9.18.13",
     "@griffel/react": "^1.5.22",
     "@swc/helpers": "^0.5.1",
-    "embla-carousel": "8.1.8",
-    "embla-carousel-autoplay": "8.1.8",
+    "embla-carousel": "8.2.0",
```

Comment:
using fixed versions of package dependencies is discouraged as it will create dupes in user-land. let's go with `^` (or `~`  if the package doesn't follow proper semver).

you'll get some errors from syncpack (known limitation of the tool), so you'll need to ignore those by adding these packages here 
<img width="549" alt="image" src="https://github.com/user-attachments/assets/35959621-cc61-45f4-86f4-f8a328cd945f">